### DISCUSSION TITLE
failed in app-store-connect to autosignin in iOS code magic

### DISCUSSION THREAD

**Discussion by Reddishetti** [codemagic-ci-cd/codemagic-docs#1473] [Q&A]

**Comment by rares-lupascu** [comment]
I do not want to be rude or anything but what kind of answer do u expect when u post something like this? I mean at least invest a bit of time to give us some context if u want us to invest time to help u ... food for thought

**Reply by Reddishetti** [reply]
initially I had added description(can see in the history :) ). This issue got fixed and  I am not sure how can I close this issue. so tried to make the title and description empty.

**Reply by Reddishetti** [reply]
Thanks for the time.
